doc/user_guide/dynamo0p3.rst

```diff
@@ -911,8 +920,8 @@ sections above:
   is known to be discontinuous and/or for when a Kernel has been written
   so that it works with fields on any of the discontinuous spaces;
 
-* ``ANY_W2`` for any type of a vector ``W2*`` function space, i.e.\ ``W2``,
-  ``W2H``, ``W2V`` and ``W2broken`` but not ``W2trace``.
+* ``ANY_W2`` for any type of a vector ``W2*`` function space, i.e. ``W2``,
```

Comment:
As said in a comment below, this is a bit complicated. There is issue #540 to resolve how `ANY_W2` metadata should be called, what `W2*` spaces should be included and whether it should be in its own list rather than just added to the continuous function space list. I left a `TODO` reference in `dynamo0p3.py`.

Reply:
Aha. Please could you add a reference to that Issue at this point in the user guide too.

Reply:
Added the reference in the user guide as a separate note.